onnxscript/rewriter/ort_fusions/_core.py

```diff
@@ -26,9 +32,32 @@
     *fused_matmul_rule_sets.fused_matmul_rule_sets(),
 ]
 
+_extra_opt_rules = llama_p0_rule_set()
```

Comment:
maybe we can make this a different structure if we want to apply multiple rule sets in the future?

For example,
`_extra_opt_rules = [llama_p0_rule_set()]`

and then loop over them,
`
for rule in _extra_opt_rules:
      rule.apply_to_model(model)
`

Reply:
Not sure I understand. I think a few things to be done are: what's called llama_p0_rule_set is actually generic, we should be adding them to the list of generic optimizations, so we won't even need this here. It should become part of the default optimizer anyway.

We may want, as a convenience, the ability to merge multiple rule sets (define a new ruleset as the union of other rulesets).